setup.py

```diff
@@ -8,5 +8,5 @@
       author_email='kyle@citrine.io',
       packages=find_packages(),
       install_requires=[
-            'six==1.10.0'
+            'six>=1.10.0'
```

Comment:
Why not cap it at 2?  If they are semantic, 2 could introduce a breaking change.